.github/workflows/test.yml

```diff
@@ -8,7 +8,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x, 15.x]
+        node-version: [16.x, 17.x, 18.x, 19.x]
```

Comment:
For node, there's not much reason to test odd number releases if there's a greater even number release, as the latter is going to be EOL, and that really nobody uses those odd number releases. Given that Node 20 is out, if we wanted to keep this like this, then would suggest to test against:

```
        node-version: [16.x, 18.x, 20.x]
```

Though given suggestion to upgrade to Node 20, would simplify this to be just:

```
        node-version: [20.x]
```

as no reason to test against old versions of node.

Reply:
Great, thank you! That's exactly what I thought. Just uploaded these changes.